config/routes.rb

```diff
@@ -1,3 +1,21 @@
 Rails.application.routes.draw do
+  get 'review/show'
```

Comment:
Should use route resources

see http://guides.rubyonrails.org/routing.html#resources-on-the-web